research/conditional/train/cc_train.py

```diff
@@ -109,6 +108,8 @@ def main(
         args.save_weights_path = os.path.abspath(args.save_weights_path)
         os.makedirs(args.save_weights_path, exist_ok=True)
 
+    param_precision = torch.bfloat16 if args.mixed_precision else torch.float32
+
```

Comment:
same as before: bfloat or float?

Reply:
I think `bfloat16` is basically just superior. But we can discuss it in the group

Reply:
yeah, this is not something that can be just sneaked into `main`

Reply:
As discussed in person, this has been discussed with the team and `bfloat` has been deemed the reduced precision of choice.

Reply:
@kuba-krj when this PR goes through, announce this on our slack channel (the bfloat16 info)